### ISSUE TITLE
Background color doesn't change when you hover over the "details" and "copy" buttons

### ISSUE THREAD

**Issue by Roberrtt** [BeamMW/beam-ui#607]
In transaction details, background color doesn't change when you hover over the "details" and "copy" buttons

Version 6.0 on the left : Version 5.3 on the right

https://user-images.githubusercontent.com/50007056/116056737-ca9c4700-a686-11eb-99c2-ec149efd6445.mp4

**Comment by chapati** [comment]
This is most like will be not releveant, since we're moving details to dialog on 6.1
Transferring to 6.1 and putting on hold until decision is made.

**Comment by chapati** [comment]
Not releveant after #590
